src/SourceBuild/content/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "8.0.100-preview.2.23157.25"
+    "dotnet": "8.0.100-preview.3.23178.1"
```

Comment:
This doesn't appear to be a valid MSFT built SDK.  This is not the same as PrivateSourceBuiltSdkUrl_CentOS8Stream.  You will need to find a MSFT built SDK from the same installer commit as you pulled the artifacts from.

Reply:
You're right. That is a source-built SDK. Will update this soon.

Reply:
Global.json change was not needed - fixed with commit: https://github.com/dotnet/installer/pull/15980/commits/c847af3b687620b209677efb6983aa2fa4124c61

Reply:
We don't want to update the artifacts without updating the SDK to equivalent MSFT SDK.  Without this it increases our chances of failures in internal CI and certainly will see differences between the various legs.

Reply:
I did check the version of SDK that was used for building the artifacts archive. It is the same as the one that was already in global.json, so there is no change in this file.

Reply:
Perhaps I did not pick the equivalent SDK version - the one I picked was in VMR's global.json file, and used for building source-build.

Reply:
Updated the PR with correct SDK version. PSB artifacts were picked up from https://dev.azure.com/dnceng/internal/_build/results?buildId=2145903&view=results. Matching Installer build is: https://dev.azure.com/dnceng/internal/_build/results?buildId=2145868&view=results